src/main/java/com/dataartisans/flinktraining/exercises/datastream_java/popular_places/PopularPlaces.java

```diff
@@ -27,6 +27,7 @@
 import org.apache.flink.api.java.tuple.Tuple4;
 import org.apache.flink.api.java.tuple.Tuple5;
 import org.apache.flink.api.java.utils.ParameterTool;
+import org.apache.flink.hadoop.shaded.com.google.common.collect.Iterables;
```

Comment:
@smarthi Yes, please use a proper Maven dependency. Otherwise this breaks Maven's dependency resolution. Since this is the only place where you use Guava, I would just leave the old code (which does actually the same as the Iterables.size(..) call).